README.md

```diff
@@ -46,7 +46,9 @@ Quickstart:
 <script src="dist/quicklink.umd.js"></script>
 <!-- Initialize (you can do this whenever you want) -->
 <script>
-quicklink.listen();
+  quicklink.listen({
+    el: document.querySelectorAll('a.linksToPrefetch'),
```

Comment:
(Don't hate me!) Let's restore the original / default behavior for this snippet :) I'm really happy for us to strongly encourage folks to set specific nodes to observe in a snippet right after this, but it would be a fundamental model shift to suggest you can't just drop in Quicklink and have it add value without some level of configuration.

Reply:
Hi Addy, I would never hate you! 
I know, that's my bad, it's a radical change, I am only worried that unskilled developers would add Quicklink as is and they put down their Server without having a CDN, hitting too much Origin/Database etc.

What about keeping the original 0 friction use case and put this new method as the "suggested" method to restrict Prefetch to specific Links?